src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java

```diff
@@ -41,7 +42,7 @@ public void toModelType_validPersonDetails_returnsPerson() throws Exception {
     public void toModelType_invalidName_throwsIllegalValueException() {
         JsonAdaptedPerson person =
                 new JsonAdaptedPerson(INVALID_NAME, VALID_PHONE, VALID_EMAIL, VALID_REMARK, VALID_TAGS,
-                        Type.APPLICANT.toString(), "resume review");
+                        Type.APPLICANT.toString(), ApplicantState.OUTCOMEONE.toString());
```

Comment:
Might just be me, but should `OUTCOMEONE` be `OUTCOME_ONE` instead so it is easier to read?